llvm/lib/FuzzMutate/IRMutator.cpp

```diff
@@ -162,8 +162,8 @@ void InstDeleterIRStrategy::mutate(Function &F, RandomIRBuilder &IB) {
   auto RS = makeSampler<Instruction *>(IB.Rand);
   for (Instruction &Inst : instructions(F)) {
     // TODO: We can't handle these instructions.
-    if (Inst.isTerminator() || Inst.isEHPad() ||
-        Inst.isSwiftError() || isa<PHINode>(Inst))
+    if (Inst.isTerminator() || Inst.isEHPad() || Inst.isSwiftError() ||
```

Comment:
Remove the white space only change?

Reply:
> Remove the white space only change?

I have to keep this or `clang-format` test will fail